rootfs/templates/wrapper

```diff
@@ -88,25 +88,40 @@ function use() {
   if [ -n "${LOCAL_HOME}" ]; then
     local_home=${LOCAL_HOME}
   elif [[ $(uname -r) =~ Microsoft$ ]]; then
-    windows_user_name=$(/mnt/c/Windows/System32/cmd.exe /c 'echo %USERNAME%' | tr -d '\r')
+    
+    wsl_config=`cat /etc/wsl.conf`
+    mount_path_regex="\[automount\](.|\n|\r)*root\s+=\s+(\S+)"
```

Comment:
it wouldn't be work, in case when root property are commented

Reply:
Instead `if else` and reading `wsl.conf` you can just get the right path from mounted file systems by file system `drvfs ` type and windows path `C:`
`dirname $(findmnt -S C: -t drvfs -no target)`